e2e/version_test.go

```diff
@@ -14,6 +14,7 @@ import (
 	"github.com/runfinch/finch/pkg/version"
 )
 
+//TODO: Ang's review
```

Comment:
The only way I can come up with now is to shell into the VM using binary `/Applications/finch/lima/bin/limactl` and run the version command, then compare with `finch version`. This would work in CI